docs/writing-tests/snapshot-testing.md

```diff
@@ -8,53 +8,173 @@ Storybook is a helpful tool for snapshot testing because every story is essentia
 
 ![Example Snapshot test](./snapshot-test.png)
 
-## Set up test runner
+## Migrating Tests
 
-Storybook test runner turns all of your stories into executable tests. It is powered by [Jest](https://jestjs.io/) and [Playwright](https://playwright.dev/).
+The Storyshots addon was the original testing solution for Storybook, offering a highly extensible API and a wide range of configuration options for testing. However, it was difficult to set up and maintain, and it needed to be compatible with the latest version of Storybook, which introduced some significant architectural changes. As a result, Storyhots is now officially deprecated, is no longer being maintained, and will be removed in the next major release of Storybook. We recommend following the migration guide we've prepared to help you during this transition period.
 
-Follow the [setup instructions](./test-runner#setup) to install and run the test runner.
+### Prerequisites
 
-Then, you can use the [test hook API](./test-runner#test-hook-api-experimental) to snapshot the HTML of each story:
+Before you begin the migration process, ensure that you have:
+
+- A fully functional Storybook configured with one of the [suppported frameworks](../configure/frameworks.md) running the latest stable version (i.e., 7.6 or higher).
+- Familiarity with your current Storybook and its testing setup.
+
+### With the test-runner
+
+Storybook test-runner turns all of your stories into executable tests. Powered by [Jest](https://jestjs.io/) and [Playwright](https://playwright.dev/). It's a standalone, framework-agnostic utility that runs parallel to your Storybook. It enables you to run multiple testing patterns in a multi-browser environment, including interaction testing with the [play function](./interaction-testing.md), DOM snapshot, and [accessibility testing](./accessibility-testing.md).
```

Comment:
This doesn't mention its limitations with React Native and workaround as [stated here](https://github.com/storybookjs/test-runner/blob/next/MIGRATION.md#option-1---storybook-test-runner). Is that on purpose?

Reply:
Yes, React Native and React Native are not mentioned as we don't have documentation for it